extensions/sd_api_pictures/script.py

```diff
@@ -102,12 +127,7 @@ def input_modifier(string):
 
     if triggers_are_in(string):  # if we're in it, check for trigger words
         toggle_generation(True)
-        string = string.lower()
-        if "of" in string:
-            subject = string.split('of', 1)[1]  # subdivide the string once by the first 'of' instance and get what's coming after it
-            string = "Please provide a detailed and vivid description of " + subject
-        else:
-            string = "Please provide a detailed description of your appearance, your surroundings and what you are doing right now"
+        string = f"\n{name2}: *Translates 3 INPUTs to image prompt format. The prompts are descriptive and ALWAYS include a detailed description of the subject of the image in the prompt. Any references in the second person like \"you\", \"your\", or \"yourself\" are about {name2} and include detailed information about {name2}'s hair, body type, clothing, surroundings, etc.*\nINPUT: send me a picture of a cool looking crystal\n{name2}: (crystal), diamond cut, reflective rock, transparent gemstone, beautiful sparkling gem, ultra detailed, volumetric lighting, professional photography\nINPUT: show me an image of Brad Pitt\n{name2}: portrait photo of (Brad Pitt), man, brown hair, short hair, athletic build, handsome, fashionable clothes, photo realistic, detailed skin, masterpiece, cinematic lighting\nINPUT: " + string
```

Comment:
This looks like a great way to get a more comprehensive pictures out of the model.

There's a downside, though. The dialogue flow is broken, it's like an instruct-injection right in the middle of the conversation. Wouldn't this contaminate future contexts for the model?

@oobabooga Is there a way to pass this to LLM hidden from the history; or can we somehow remove it once the response is made?